tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -445,7 +445,7 @@ procSuite "WakuNode - RLN relay":
       proofAdded1 = node1.wakuRlnRelay.appendRLNProof(wm1, time)
       # another message in the same epoch as wm1, it will break the messaging rate limit
       wm2 = WakuMessage(payload: "message 2".toBytes(), contentTopic: contentTopic)
-      proofAdded2 = node1.wakuRlnRelay.appendRLNProof(wm2, time + EpochUnitSeconds)
+      proofAdded2 = node1.wakuRlnRelay.appendRLNProof(wm2, time)
```

Comment:
@rymnc Unsure I understand why `wm2` had `time + EpochUnitSecods` if its expected to "break the messaging rate limit". Isn't it better to set the same time?

Reply:
doesn't matter too much, just showing that 2 messages were generated within the same epoch window.